test/functional/test-shadow-embed.js

```diff
@@ -93,4 +94,28 @@ describe('createShadowEmbedRoot', () => {
     const shadowRoot = createShadowEmbedRoot(hostElement, ['amp-ext1']);
     expect(savedShadowRoot).to.equal(shadowRoot);
   });
+
+  describe('scopeShadowCss', () => {
+
+    function scope(css) {
+      return scopeShadowCss(hostElement, css).replace(/[\n\t\n]/g, '');
+    }
+
+    it('should replace root selectors', () => {
+      expect(scope('html {}')).to.equal('#h amp-html {}');
```

Comment:
We could. But tokenizer is rule-based, so they are never really together when rewritten. But I'll add one or two additional asserts.